### PR TITLE
Practice app/feature/forum

### PR DESCRIPTION
**Pull request by demet47**
Added .yml files. Practice-app for features concerning discussion forum.

### CODE REVIEW

**Comment by azizamankenova** [comment]
Hi Demet! Thanks for your work!
In the discussion_forum.py file you assign the creator to the user's email, but it should store the id of the user because it was defined as integer in models.py
>creator = current_user.email

**Comment by azizamankenova** [comment]
Also, when creating a new post, nothing happens pressing "post to forum". It would be better if you could redirect to the Forum Page upon successful creation of the post, as you can not tell if the new post was created or not after pressing the button.

**Comment by demet47** [comment]
Thanks for the feedback! I fixed the database part and added redirection to discussion forum page.

File: practice-app/website/api/discussion_forum.py [hunk 1]
```diff
@@ -34,18 +34,26 @@ def bad_word_check(body):
 @forum.route('/forum_get/', methods=["GET"])
 def forum_get():
 
+    """
+    file: ./doc/forum_get_GET.yml
```
Comment (canatakan) [inline practice-app/website/api/discussion_forum.py:38]: I think you can rename these files as `forum_POST.yml` and `forum_GET.yml` for simplicity.

**Review by canatakan — CHANGES_REQUESTED** [review]
Please see my notes about the documentation before merging. Apart from those, the functionality seems okay.

File: practice-app/website/test/test_forum.py [hunk 2]
```diff
@@ -3,9 +3,12 @@
...
 
+
 from website import create_app, db
 import unittest
 
+TEST_DB_NAME = "test_database.db"
+
 class TestForum(unittest.TestCase):
     def setUp(self):
         app = create_app(testing=True)
```
Comment (azizamankenova) [inline practice-app/website/test/test_forum.py:14]: you have a typo in Line 27, it should have been "content_uri" 
> "conent_uri": "https://www.hepsiburada.com/hayatburada/wp-content/uploads/2021/10/shutterstock_1024133086.jpg"

File: practice-app/website/test/test_forum.py [hunk 3]
```diff
@@ -3,9 +3,12 @@
 sys.path.append('./website') #to run the test without import errors, one should call from one directory above this line's directory
 sys.path.append('.')
```
Comment (azizamankenova) [inline practice-app/website/test/test_forum.py:5]: Unittest cases look good, I have just a suggestion to make unit tests for GET requests, too.

Comment (demet47) [reply practice-app/website/test/test_forum.py:5]: I skipped that since it required changes in the api/discussion_forum.py file.

**Review by canatakan — APPROVED** [review]

**Comment by azizamankenova** [comment]
@demet47 Thanks for your updates! Everything looks fine, ready for merging.

**Pull request merged**